### ISSUE TITLE
shuffle=False in all Dataloaders

### ISSUE THREAD

**Issue by DarrenIm** [Haochen-Wang409/U2PL#42]
Hi, great work! But I just found out that all dataloaders(sup or unsup) are all set to shuffle=False during training. Can you explain why?
For example, in pascal_voc.py code 116 ~ 157
```      
        data_list_unsup = cfg["data_list"].replace("labeled.txt", "unlabeled.txt")
        dset_unsup = voc_dset(
            cfg["data_root"], data_list_unsup, trs_form_unsup, seed, n_sup, split
        )

        sample_sup = DistributedSampler(dset)
        loader_sup = DataLoader(
            dset,
            batch_size=batch_size,
            num_workers=workers,
            sampler=sample_sup,
            shuffle=False,
            pin_memory=True,
            drop_last=True,
        )

        sample_unsup = DistributedSampler(dset_unsup)
        loader_unsup = DataLoader(
            dset_unsup,
            batch_size=batch_size,
            num_workers=workers,
            sampler=sample_unsup,
            shuffle=False,
            pin_memory=True,
            drop_last=True,
        )
        return loader_sup, loader_unsup
```
Thanks in advance.

**Comment by Haochen-Wang409** [comment]
Hi, thanks for your attention of our work.

In fact, we simply follow AEL, which is previous state-of-the-art, where ```shuffle=False``` for all loaders.
See https://github.com/hzhupku/SemiSeg-AEL/blob/main/semseg/dataset/pascal_voc_semi_cp.py.

Maybe you could try switching ```shuffle=True```.
